docs/topics_5/web-review-reports.adoc

```diff
@@ -5,7 +5,7 @@
 
 The {ProductShortName} {WebName} provides a set of detailed reports that can help you decide if you need to make any changes to your applications. You access these reports from the *Analysis results* screen.
 
-The reports are described in detail in link:{ProductDocUserGuideURL}/index#review_reports_cli-guide/[{ProductShortName} reports] in the _{UserCLIBookName}_.
+The reports are described in detail in link:{ProductDocUserGuideURL}/index#review_reports_cli-guide[{ProductShortName} reports] in the _{UserCLIBookName}_.
```

Comment:
I think this is better and more streamlined:

The reports are described in detail in the link:{ProductDocUserGuideURL}/index#review_reports_cli-guide[{UserCLIBookName}].

The user does not need to know the name of the section that the link points to. It's sufficient to say that they can find more info on reports in the CLI guide, with the link pointing to the reports section.